packages/apollo-gateway/src/datasources/RemoteGraphQLDataSource.ts

```diff
@@ -87,9 +91,10 @@ export class RemoteGraphQLDataSource implements GraphQLDataSource {
     >,
   ): ValueOrPromise<void>;
 
-  public async didReceiveResponse<TResult = any>(
+  public async didReceiveResponse<TContext, TResult = any>(
```

Comment:
Fly-by comment: Adding this as the first positional type argument would be a breaking change for any existing implementors (who aren't us, since we just rely on the default of `any`) — can it be added after `TResult = any` with its own default value?

(Realistically though, we should probably have this `TContext` as a type variable on the entire class rather than having it localize to this method.)

Reply:
You're absolutely right. I took the easy way in 0ecd617.

I'll take another stab at `TContext` on the class, but `RemoteGraphQLDataSource.process()` wasn't really having it.